content/docs/rfcs/43/README.md

```diff
@@ -0,0 +1,354 @@
+
+The reason for such suggestion is due to the fact that if an attacker is able to compromise one of the ephemeral keys, 
+he might successfully realize an undetected MitM attack up to the `authcode` confirmation 
+(we note that compromising ephemeral keys is outside our and Noise security assumptions).
+
+The attacker could indeed proceed as follows:
+- intercepts the QR;
+- blocks/delays the delivery of the pairing message `b.`;
+- compromises `A` or `B` ephemeral key;
+- recovers the genuine `authcode` that would have been generated by `A` and `B`;
+- generates ~`10^8` random `t` values until the Noise processing of the message `b'. -> eC, eCeB  {H(sC||t)} `, where `eC` and `sC` are the attacker ephemeral and static key, respectively, results in computing the same `authcode` as the one between `A` and `B`; 
+- delivers the message `b'. -> eC, eCeB  {H(sC||t)}` to `B` (before `A` is able to deliver its message `b.`).
+
+At this point `A` and `B` will observe the same `authcode` (and would then confirm it),
+but `B` will process the attacker's ephemeral key `eC` instead of `eA`. 
+
+However, the attacker would not be able to open to device `A` the static key commitment `H(sB||s)` sent by device `B` out-of-band, 
+and the pairing will abort on `A` side before it reveals its static key. 
+Device `B`, instead, will successfully complete the pairing with the attacker.
+
+Hence, timeboxing the QR exposure,
+also in combination with increasing the number of decimal digits of the `authcode`,
+will strongly limit the probability that an attacker can successfully impersonate device `A` to `B`.
+
+We stress once more, that such attack requires the compromise of an ephemeral key (outside our security model)
+and that device `A` will in any case detect a mismatch and abort the pairing, 
+regardless of the fact that the QR timeboxing mitigation is implemented or not.
+
+## Randomized Rekey
+
+The Noise Protocol framework supports [`Rekey()`](http://www.noiseprotocol.org/noise.html#rekey) 
+in order to update encryption keys *"so that a compromise of cipherstate keys will not decrypt older* \[exchanged\] *messages"*. 
+However, if a certain cipherstate key is compromised, 
+it will be possible for the attacker not only to decrypt messages encrypted under that key, 
+but also all those messages encrypted under any successive new key obtained through a call to `Rekey()`.
+
+This can be mitigated by:
+- keeping the full Handhshake State even after the handshake is complete (*by Noise specification a call to [`Split()`](http://www.noiseprotocol.org/noise.html#the-symmetricstate-object) should delete the Handshake State*)
+- continuing updating the Handshake State by processing every after-handshake exchanged message (i.e. the `payload`) according to the Noise [processing rules](http://www.noiseprotocol.org/noise.html#processing-rules) (i.e. by calling `EncryptAndHash(payload)` and `DecryptAndHash(payload)`);
```

Comment:
I suppose `payload` is unknown to the attacker and that is why the successive keys remain hidden from the attacker, right?